apps/vaporgui/PFidelitySection.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include "vapor/RenderParams.h"
+
+#include "PSection.h"
+#include "PLineItem.h"
+#include "PWidgetHLI.h"
+
+class VComboBox;
+
+class PFidelitySection : public PSection {
+public:
+    PFidelitySection();
+};
+
+class PQuickFidelitySelector : public PLineItem {
+    Q_OBJECT
+    VComboBox *_vComboBox;
+public:
+    PQuickFidelitySelector();
+
+protected:
+    virtual void updateGUI() const override;
+    bool requireDataMgr() const override { return true; }
+    
+private slots:
+    void dropdownTextChanged(std::string text);
+};
+
+class PLODSelector : public PLineItem {
+    Q_OBJECT
+    VComboBox *_vComboBox;
+public:
+    PLODSelector();
+
+protected:
+    virtual void updateGUI() const override;
+    bool requireDataMgr() const override { return true; }
+    
+private slots:
+    void dropdownIndexChanged(int i);
+};
+
+class PRefinementSelector : public PLineItem {
+    Q_OBJECT
+    VComboBox *_vComboBox;
+public:
+    PRefinementSelector();
+
+protected:
+    virtual void updateGUI() const override;
+    bool requireDataMgr() const override { return true; }
+    
+private slots:
+    void dropdownIndexChanged(int i);
+};
+
+class PLODSelectorHLI :
```

Comment:
This appears to be dead code. This class isn't used anywhere. Note that PLODSelector::dropdownIndexChanged() is already using the Params HLI

Reply:
Removed